app/models/night_finder.rb

```diff
@@ -0,0 +1,30 @@
+class NightFinder
+  def find_nights(night_params)
+    @_night_params = night_params
```

Comment:
@njgheorghita just trying out the convention Josh mentioned in class. It tells the reader that it is only accessed in a private method.